app/models/user.rb

```diff
@@ -11,4 +11,5 @@ class User < ActiveRecord::Base
   validates :username, uniqueness: true
   validates :username, presence: true
   validates :city, presence: true
+  validates :admin, :inclusion => {:in => [true, false]}
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.